modules/core/src/v2/wallet.ts

```diff
@@ -167,6 +167,23 @@ export class Wallet {
     }).call(this).asCallback(callback);
   }
 
+  /**
+   * Gets keys for signing
+   * @param params
+   * @param callback
+   * @returns {Wallet}
```

Comment:
Doesn't look like we return a wallet object.

Reply:
fixed